include/kodlab_mjbots_sdk/robot_base.h

```diff
@@ -51,7 +48,7 @@ namespace kodlab
                 joints.push_back(j); // Copy constructed, uses an implicit upcasting
                 positions_.push_back( j->get_position_reference() );
                 velocities_.push_back( j->get_velocity_reference() );
-                torque_cmd_.push_back( j->get_servo_torque_reference() );
+                torque_cmd_.push_back(j->get_torque_cmd_reference() );
```

Comment:
Please match the spacng one way or the other

Reply:
done